frb_example/pure_dart_multi/dart/lib/main.dart

```diff
@@ -0,0 +1,435 @@
+import 'dart:ffi';
+import 'dart:typed_data';
+
+import 'package:flutter_rust_bridge/flutter_rust_bridge.dart';
+import 'bridge_generated_api_1.dart';
+import 'bridge_generated_api_2.dart';
+import 'package:test/test.dart';
+
+void main(List<String> args) async {
+  String dylibPath1 = args[0];
+  String dylibPath2 = args[1];
+  print(
+      'flutter_rust_bridge example program start (dylibPath=$dylibPath1,$dylibPath2)');
+  print('construct api');
+  final dylib1 = DynamicLibrary.open(dylibPath1);
+  final api1 = ApiClass1Impl(dylib1);
+
+  test('dart call simpleAdder', () async {
```

Comment:
No need to repeat *all* tests in `pure_dart`. Only run two tests here may be sufficient, one for api1, one for api2

Reply:
Fixed in 3917234ba483bcd16ba702640a8d4b806d27d912.